File: Lenguaje/Codeforces/Round 784 Div 4/A_Division/A_Division.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        while(t>0){
            t--;
            int n = in.nextInt();
            if(n>=1900)System.out.println("Division 1");
            else if(n>=1600)System.out.println("Division 2");
            else if(n>=1400)System.out.println("Division 3");
            else System.out.println("Division 4");
        }
    }
}

File: Lenguaje/Codeforces/Round 784 Div 4/A_Division/A_Division.c
#include <stdio.h>
int t,n;
int main(){
    scanf("%d",&t);
    while(t--){
        scanf("%d",&n);
        if(n>=1900)printf("Division 1\n");
        else if(n>=1600)printf("Division 2\n");
        else if(n>=1400)printf("Division 3\n");
        else printf("Division 4\n");
    }
    return 0;
}
